nautobot/core/templates/inc/nav_menu.html

```diff
@@ -60,9 +60,8 @@
                                                             Use 'url xxx as variable' so that an invalid
                                                             link doesn't throw a NoReverseMatch exception.
                                                         {% endcomment %}
-                                                        {% url item_link as item_url %}
```

Comment:
Is this no longer needed? If so does that make the above comment also irrelevant?